packages/@aws-cdk/assets/test/fs/test.fs-fingerprint.ts

```diff
@@ -68,11 +68,13 @@ export = {
       const outdir = fs.mkdtempSync(path.join(os.tmpdir(), 'copy-tests'));
       libfs.copyDirectory(srcdir, outdir);
 
+      const options = { exclude: ['*.ignoreme'] };
```

Comment:
Please avoid modifying existing tests. Instead, add a new test that will verify the new behavior.

Reply:
This test will fail if they don't have the same `exclude` options since 
`const hashSrc = libfs.fingerprint(srcdir);` and
`const hashCopy = libfs.fingerprint(outdir, { exclude: ['*.ignoreme'] });` 
 aren't going to be equal.